tests/integration/conftest.py

```diff
@@ -743,3 +746,31 @@ async def instance_helper_fixture(request: pytest.FixtureRequest) -> OpenStackIn
     """Instance helper fixture."""
     openstack_connection = request.getfixturevalue("openstack_connection")
     return OpenStackInstanceHelper(openstack_connection=openstack_connection)
+
+
+@pytest.fixture(scope="module")
+def juju(request: pytest.FixtureRequest, model: Model) -> Generator[jubilant.Juju, None, None]:
```

Comment:
lets start using jubilant. stole this fixture from https://github.com/canonical/paas-charm ;-)